spring-boot/admin/src/main/resources/templates/admin/dashboard.html

```diff
@@ -145,6 +145,10 @@
                 updateGatewayHealthMonitorWithLatestDataFromAdmin();
             });
 
+            setInterval(function(){
+                updateGatewayHealthMonitorWithLatestDataFromAdmin()
+            }, 30000)
```

Comment:
Please put a comment to explain the "magic number".

Reply:
Done